pkgs/stdenv/generic/setup.sh

```diff
@@ -302,9 +302,9 @@ runHook addInputsHook
 
 # Recursively find all build inputs.
 findInputs() {
-    local pkg="$1"
-    local var="$2"
-    local propagatedBuildInputsFile="$3"
+    local pkg="$1"; shift
+    local var="$1"; shift
```

Comment:
Copy-paste error?

Reply:
No, he `shift`s arguments to use `$@` later.